### PR TITLE
Update free-programming-books-ar.md

### PR DESCRIPTION
**Pull request by Mo3az2000**
Added AVR section

## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [ ] Search for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by davorpa** [comment]
> It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category. "AVR" is awfully specific, if this can't go in Arduino or r-Pi, maybe make a "hardware" category?

What about homogenize like in other lists?
- https://github.com/EbookFoundation/free-programming-books/search?q=embedded
- https://github.com/EbookFoundation/free-programming-books/search?q=rtos

```md
- Embedded Systems
  - Arduino
```

**Comment by eshellman** [comment]
let's use existing "Arduino" and let someone who's read it make the correction if it's wrong

File: books/free-programming-books-ar.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 * [Arduino](#arduino)
 * [Artificial Intelligence](#artificial-intelligence)
+* [AVR](#AVR)
```
Comment (davorpa) [inline books/free-programming-books-ar.md:5]: Moving to Arduino...

```suggestion
```

File: books/free-programming-books-ar.md [hunk 2]
```diff
@@ -33,6 +34,11 @@
 * [مدخل إلى الذكاء الاصطناعي وتعلم الآلة](https://academy.hsoub.com/files/17-%D9%85%D8%AF%D8%AE%D9%84-%D8%A5%D9%84%D9%89-%D8%A7%D9%84%D8%B0%D9%83%D8%A7%D8%A1-%D8%A7%D9%84%D8%A7%D8%B5%D8%B7%D9%86%D8%A7%D8%B9%D9%8A-%D9%88%D8%AA%D8%B9%D9%84%D9%85-%D8%A7%D9%84%D8%A2%D9%84%D8%A9/)
```
Comment (davorpa) [inline books/free-programming-books-ar.md:34]: Moving to Arduino...

```suggestion
* [AVR ببساطة: من تشغيل دايود ضوئي إلى أنظمة الوقت الحقيقي](https://github.com/abdallah-ali-abdallah/Simply-AVR-Book) - عبدالله علي عبدالله (ODT, PDF)
* [مدخل إلى الذكاء الاصطناعي وتعلم الآلة](https://academy.hsoub.com/files/17-%D9%85%D8%AF%D8%AE%D9%84-%D8%A5%D9%84%D9%89-%D8%A7%D9%84%D8%B0%D9%83%D8%A7%D8%A1-%D8%A7%D9%84%D8%A7%D8%B5%D8%B7%D9%86%D8%A7%D8%B9%D9%8A-%D9%88%D8%AA%D8%B9%D9%84%D9%85-%D8%A7%D9%84%D8%A2%D9%84%D8%A9/)
```

File: books/free-programming-books-ar.md [hunk 3]
```diff
@@ -33,6 +34,11 @@
...
 
 
+### AVR
+
+* [AVR ببساطة: من تشغيل دايود ضوئي إلى أنظمة الوقت الحقيقي](https://github.com/abdallah-ali-abdallah/Simply-AVR-Book) - عبدالله علي عبدالله (ODT, PDF)
+
+
```
Comment (davorpa) [inline books/free-programming-books-ar.md:41]: Moving to Arduino...

```suggestion
```

File: books/free-programming-books-ar.md [hunk 4]
```diff
@@ -30,6 +30,7 @@
 
 ### Artificial Intelligence
 
+* [AVR ببساطة: من تشغيل دايود ضوئي إلى أنظمة الوقت الحقيقي](https://github.com/abdallah-ali-abdallah/Simply-AVR-Book) - عبدالله علي عبدالله (ODT, PDF)
```
Comment (davorpa) [inline books/free-programming-books-ar.md:33]: Moving AVR to Arduino...

```suggestion
* [AVR ببساطة: من تشغيل دايود ضوئي إلى أنظمة الوقت الحقيقي](https://github.com/abdallah-ali-abdallah/Simply-AVR-Book) - عبدالله علي عبدالله (ODT, PDF)


### Artificial Intelligence

```

**Review by davorpa — CHANGES_REQUESTED** [review]

File: books/free-programming-books-ar.md [hunk 5]
```diff
@@ -27,6 +27,8 @@
 * [أردوينو السهل](http://librebooks.org/simply-arduino/)
 * [كتاب احترف الأردوينو](http://www.ev-center.com/uploads/2/1/2/6/21261678/arduino.pdf) (PDF)
 
+* [AVR ببساطة: من تشغيل دايود ضوئي إلى أنظمة الوقت الحقيقي](https://github.com/abdallah-ali-abdallah/Simply-AVR-Book) - عبدالله علي عبدالله (ODT, PDF)
```
Comment (davorpa) [inline books/free-programming-books-ar.md:30]: Moving AVR to Arduino. Alphabetize

```suggestion
* [AVR ببساطة: من تشغيل دايود ضوئي إلى أنظمة الوقت الحقيقي](https://github.com/abdallah-ali-abdallah/Simply-AVR-Book) - عبدالله علي عبدالله (ODT, PDF)
* [أردوينو السهل](http://librebooks.org/simply-arduino/)
* [كتاب احترف الأردوينو](http://www.ev-center.com/uploads/2/1/2/6/21261678/arduino.pdf) (PDF)
```

**Review by davorpa — APPROVED** [review]

**Comment by davorpa** [comment]
Workflows off since https://github.com/EbookFoundation/free-programming-books/pull/6426/commits/185cd4e86189a33f6edf6958f8b45387f48a4949 :worried:

**Pull request merged**